includes/admin/class-wc-payments-admin.php

```diff
@@ -836,4 +837,25 @@ public function maybe_redirect_to_onboarding() {
 
 		$this->account->redirect_to_onboarding_page();
 	}
+
+	/**
+	 * Add woopay as a payment method to the edit order on admin.
+	 *
+	 * @param int $order_id order_id.
+	 */
+	public function show_woopay_payment_method_name_admin( $order_id ) {
+		$order = new WC_Order( $order_id );
```

Comment:
Here, I'm thinking we may want to use [`wc_get_order`](https://woocommerce.github.io/code-reference/files/woocommerce-includes-wc-order-functions.html#function_wc_get_order) instead, so that it will return `false` instead of throwing an exception if the order doesn't exist. Then we can also add a check for `! $order` to return early in that case too.

Reply:
Fixed with 0b5b5e2383e41972e8e3f3546416d4acef915bea